csvGenerator.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "6f275807",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import csv\n",
+    "import math\n",
+    "import random\n",
+    "import requests\n",
+    "from CitesphereConnectorTest import *\n",
+    "from CitesphereConnector import CitesphereConnector"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "2c0597b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth_object = authObject()\n",
```

Comment:
ah. two things:
- class names are capitalized, so it needs to be `AuthObject` (I know you didn't write it but you should fix it)
- and since this class seems to be a general one, not just for tests it should not be in the test file

Reply:
created a separate file for the AuthObject Class